code/modules/clothing/under/accessories/permits.dm

```diff
@@ -92,3 +107,9 @@
 	icon = 'icons/obj/storage.dmi'
 	icon_state = "permit"
 	starts_with = list(	/obj/item/clothing/accessory/permit/gun = 8)
+
+/obj/item/weapon/storage/box/gun_permits/New()
+	for(var/instance in subtypesof(/obj/item/clothing/accessory/permit/gun))	// gives one of each permit, in a box.
+		starts_with += instance[1]
```

Comment:
This works?

Reply:
Doesn't. It's removed in the next commit that will be published tonight.